articles/data-explorer/data-connection-event-grid-csharp.md

```diff
@@ -90,4 +90,38 @@ await kustoManagementClient.DataConnections.CreateOrUpdateAsync(resourceGroupNam
 | consumerGroup | *$Default* | The consumer group of your Event Hub.|
 | location | *Central US* | The location of the data connection resource.|
 
+## Generate sample data
+
+Now that Azure Data Explorer and the storage account are connected, you can create sample data and upload it to the blob storage.
+
+This script creates a new container in your storage account, uploads an existing file (as a blob) to that container, and then lists the blobs in the container.
+
+```csharp
+var azureStorageAccountConnectionString=<storage_account_connection_string>;
+
+var containerName=<container_name>;
+var blobName=<blob_name>;
+var localFileName=<file_to_upload>;
+
```

Comment:
I suggest to keep the same style of coding and doc, like var containerName = "mycontainer"...
string storageConnectionString = "DefaultEndpointsProtocol=https;AccountName=xxxxxxxxxxxxxx;AccountKey=xxxxxxxxxxxxxx;EndpointSuffix=core.windows.net";
..
Add table at the end to explain what's the value

Reply:
It might be better to expose code that doesn't compile, forcing users to change each property. Otherwise changes are accidentally missed.